lib/devise/jwt.rb

```diff
@@ -58,6 +58,10 @@ def self.forward_to_warden(setting, value)
             default: Warden::JWTAuth.config.revocation_requests,
             constructor: ->(value) { forward_to_warden(:revocation_requests, value) })
 
+    setting(:token_header,
+            default: Warden::JWTAuth.config.token_header,
+            constructor: ->(value) { forward_to_warden(:token_header, value) })
```

Comment:
Seems like no tests are required for this change, please correct me if I'm wrong